migrate.py

```diff
@@ -1205,6 +1198,28 @@ def assemble_collections(checkout_path, spec, args, target_github_org):
             REMOVE = set()
 
 
+def rewrite_unit_tests(collection_dir, collection, spec, namespace, args):
+    deps = []
+
+    for file_path in itertools.chain.from_iterable(
+            (os.path.join(dp, f) for f in fn if f.endswith('.py'))
+            for dp, dn, fn in os.walk(os.path.join(collection_dir, 'tests', 'unit'))
+    ):
+        _unit_test_module_src_text, unit_test_module_fst = read_module_txt_n_fst(file_path)
+        deps += rewrite_imports(unit_test_module_fst, collection, spec, namespace, args)
```

Comment:
It's cleaner to `yield` things here and then `list()` outside...
Like
```suggestion
        yield rewrite_imports(unit_test_module_fst, collection, spec, namespace, args)
```